src/github/client/github-client.ts

```diff
@@ -88,8 +88,11 @@ export default class GitHubClient {
 	 * API in the name of an installation of that app (to access the users' data).
 	 */
 	private async createInstallationToken(githubInstallationId: number): Promise<AuthToken> {
-		const response = await this.axios.post<Octokit.AppsCreateInstallationTokenResponse>(`/app/installations/${githubInstallationId}/access_tokens`, {}, {
-			...this.appAuthenticationHeaders()
+		const response = await this.axios.post<Octokit.AppsCreateInstallationTokenResponse>(`/app/installations/{githubInstallationId}/access_tokens`, {}, {
+			...this.appAuthenticationHeaders(),
+			urlParams: {
+				githubInstallationId
+			}
```

Comment:
That's the main fix. Thankfully we have tons of tests which already cover this path